compiler/src/main/scala/org/scalajs/core/compiler/GenJSCode.scala

```diff
@@ -1255,7 +1305,8 @@ abstract class GenJSCode extends plugins.PluginComponent
         if (scalaPrimitives.isPrimitive(sym) &&
             !jsPrimitives.shouldEmitPrimitiveBody(sym)) {
           None
-        } else if (isAbstractMethod(dd)) {
+        } else if (isAbstractMethod(dd) && !(scalaUsesImplClasses &&
+            jsInterop.isSymbolForwarder(sym))) {
```

Comment:
Is `scalaUsesImplClasses &&` necessary here?
If yes, can you put it on the next line, so that the line break goes with the outer-most `&&` rather than the inner-most one?